app/controllers/assessment/gradings_controller.rb

```diff
@@ -195,11 +195,14 @@ def show
 
   private
 
-  # Compare answers against attempts, modulo case and punctuation
+  def normalise(answer)
+    answer.downcase.gsub(/[^\w\d\s]+/, ' ').gsub(/\s+/, ' ').strip
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.